<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ElementLog</class>
 <widget class="QDialog" name="ElementLog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="SubmitBtn">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>256</width>
     <height>83</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>元素所在行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="RowNum"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>元素所在列</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ColNum"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>所要更新值</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="ValNum"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
